packages/compass-web/src/index.tsx

```diff
@@ -152,6 +163,40 @@ const CompassWeb = ({
   const [connectionError, setConnectionError] = useState<any | null>(null);
   const dataService = useRef<DataService>();
 
+  const loggerProvider = useRef({
+    createLogger: (component = 'SANDBOX-LOGGER'): LoggerAndTelemetry => {
+      const logger = (name: 'debug' | 'info' | 'warn' | 'error' | 'fatal') => {
+        return (...args: any[]) => {
+          logging.push({ name, component, args });
```

Comment:
I'm storing the logs too, but we don't use that in the e2e tests yet. We could save that out when we cleanup compass and upload to evergreen, but it would only be the renderer's logs and probably rarely the useful logging?

Reply:
Nevermind it was quick so I just added it.